src/transformers/models/bert/modeling_tf_bert.py

```diff
@@ -722,8 +781,8 @@ def call(
         # positions we want to attend and -10000.0 for masked positions.
         # Since we are adding it to the raw scores before the softmax, this is
         # effectively the same as removing these entirely.
-        extended_attention_mask = tf.cast(extended_attention_mask, embedding_output.dtype)
-        extended_attention_mask = (1.0 - extended_attention_mask) * -10000.0
+        extended_attention_mask = tf.cast(x=extended_attention_mask, dtype=embedding_output.dtype)
```

Comment:
I would remove the `x=` that is not useful here, but leave the `dtype=` which is informative.